llama_index/vector_stores/qdrant.py

```diff
@@ -49,9 +54,17 @@ class QdrantVectorStore(BasePydanticVectorStore):
     batch_size: int
     prefer_grpc: bool
     client_kwargs: dict = Field(default_factory=dict)
+    enable_hybrid: bool
 
     _client: Any = PrivateAttr()
     _collection_initialized: bool = PrivateAttr()
+    _sparse_doc_fn: Optional[SparseEncoderCallable] = PrivateAttr()
+    _sparse_query_fn: Optional[SparseEncoderCallable] = PrivateAttr()
+    _hybrid_fusion_fn: Optional[
```

Comment:
should we give some default fusion func as well

Reply:
the default is the deduplicate function